inspirehep/utils/tickets.py

```diff
@@ -24,26 +24,222 @@
 
 from __future__ import absolute_import, division, print_function
 
-from flask import current_app
-from rt import Rt, ConnectionError
+from os.path import join
+
+from flask import render_template
+
+from rt import ALL_QUEUES, ConnectionError
+
+from .proxies import rt_instance
 
 
 def get_instance():
-    """Make a RT instance and return it."""
-    url = current_app.config.get("CFG_BIBCATALOG_SYSTEM_RT_URL", "")
-    login = current_app.config.get("CFG_BIBCATALOG_SYSTEM_RT_DEFAULT_USER", "")
-    password = current_app.config.get("CFG_BIBCATALOG_SYSTEM_RT_DEFAULT_PWD", "")
-
-    if url:
-        tracker = Rt(
-            url=url,
-            default_login=login,
-            default_password=password,
-        )
-        tracker.login()
-        return tracker
+    return rt_instance
 
 
 def retry_if_connection_problems(exception):
     """Return True if exception is an ConnectionError in Rt."""
     return isinstance(exception, ConnectionError)
+
+
+def create_ticket(queue, requestors, subject, body, recid=None, **kwargs):
+    """Creates new RT ticket and returns new ticket id.
+
+    :param queue: queue name for the ticket
+    :type queue: string
+
+    :param requestors: username to set to requestors field of the ticket
```

Comment:
yep, that's an RT thing, and it's a bit weird. Actually their rest documentation is also a bit mixed, sometimes it's called `Requestor` but usually `Requestors` :S